webapp/packages/plugin-connections/src/ConnectionForm/IConnectionFormState.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2025 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+import { schema } from '@cloudbeaver/core-utils';
+import { CONNECTION_CONFIG_SCHEMA } from './Options/IConnectionConfig.js';
+import type { IFormProps } from '@cloudbeaver/core-ui';
+
+export const CONNECTION_FORM_STATE_SCHEMA = schema
+  .object({
+    projectId: schema.string(),
+    config: CONNECTION_CONFIG_SCHEMA,
```

Comment:
I will not recommend using this property (`config`) here, instead, when you needed you can always access the state of a part to change the initial state or state of it